contracts/hackatom/tests/integration.rs

```diff
@@ -191,6 +197,7 @@ fn failed_handle() {
 }
 
 #[test]
+#[cfg(feature = "singlepass")]
```

Comment:
I would move all the `#[cfg(feature = "singlepass")]` gated `use` statements above into this function. If this is the only place we use that code and requires singlepass. If there are more places, then good as it is.

(And yes, you can `use` in a function. It imports at compile time, but scopes the imports to the surrounding function.)

Reply:
Good idea. In d7f8b048e148b8606ed8b71a450518f1125868d6 I created a separate scope for that, which I think is easier to understand and ready for multiple singlepass tests:

```rust
#[cfg(feature = "singlepass")]
mod singlepass_tests {
    use super::*;

    use cosmwasm::serde::to_vec;
    use cosmwasm_vm::call_handle;
    use cosmwasm_vm::testing::mock_instance_with_gas_limit;

    #[test]
    fn handle_panic_and_loops() {
      // ..
```